src/TaylorInversion.jl

```diff
@@ -111,13 +102,14 @@ function further_substitution(it::InverseTaylor{N}, subbed::Num) where {N}
     return subbed
 end
 
-function create_expressions(n::Int)
-    it = InverseTaylor{n}()
+@generated function (::TaylorInverter{N})(var"ˍ₋arg1") where {N}
```

Comment:
Is it _necessary_ to switch from a regular function to a callable struct? If so, why exactly? And if not, why do you choose to do so anyway?

Reply:
It's not necessary, but it's the easiest way to get around to it.